### PR TITLE
PR-template-added

### PR DESCRIPTION
**Pull request by vish-han**
I have added the PR template which was not there previously , Please review the code and let me know if u want some changes✌️😊.

### CODE REVIEW

**Comment by cnu1812** [comment]
You worked under wrong directory. Change the directory else template won't work.

**Comment by vish-han** [comment]
Oh sorry my bad , can u please let me know in which directory I should add this file😃

**Review by subhamgcon — CHANGES_REQUESTED** [review]
Take the code out of the workflow folder and make another folder as PULL_RUQUEST_TEMPLATE and then push the md file in that folder to make it work

File: .github/workflows/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+
+# Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] My code is not pirated 
+- [ ] I have performed a self-review of my code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
```
Comment (AbhijithGanesh) [inline .github/workflows/PULL_REQUEST_TEMPLATE/pull_request_template.md:26]: This is in the wrong directory, `PULL_REQUEST_TEMPLATE/` is the correct directory to have the template. Please make the changes :)

**Review by AbhijithGanesh — CHANGES_REQUESTED** [review]

**Comment by gullapalli9482** [comment]
Is there any pending work to be done?

**Comment by cnu1812** [comment]
> Is there any pending work to be done?

Has to change the working directory

**Comment by vish-han** [comment]
@cnu1812  im making changes rn

**Comment by vish-han** [comment]
@AbhijithGanesh Can u please review it  now. please.?

**Review by AbhijithGanesh — APPROVED** [review]
LGTM

**Review by cnu1812 — APPROVED** [review]

**Pull request merged**